server/src/common/diagnostic.ts

```diff
@@ -1,11 +1,13 @@
 /*
-* diagnostics.ts
```

Comment:
The header comments (both content and formatting) are consistent across the entire pyright code base. I'd prefer that we leave this as is to maintain that consistency. If you want to change it, let's discuss and then push the change throughout the entire code base.

Reply:
I'll reformat with `prettier` formatter that Python extension uses if you don't mind.

Reply:
OK, let's do that as part of a separate PR since it will touch pretty much every file.

Reply:
I did it manually. Extension settings is 2 spaces for indent and there is more. Too many changes.